tests/actions/PolicyTest.js

```diff
@@ -0,0 +1,189 @@
+import _ from 'lodash';
+import Onyx from 'react-native-onyx';
+import CONST from '@src/CONST';
+import OnyxUpdateManager from '../../src/libs/actions/OnyxUpdateManager';
+import * as Policy from '../../src/libs/actions/Policy';
+import ONYXKEYS from '../../src/ONYXKEYS';
+import * as TestHelper from '../utils/TestHelper';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+
+const ESH_EMAIL = 'eshgupta1217@gmail.com';
+const ESH_ACCOUNT_ID = 1;
+
+OnyxUpdateManager();
+describe('actions/Policy', () => {
+    beforeAll(() => {
+        Onyx.init({
+            keys: ONYXKEYS,
+        });
+    });
+
+    beforeEach(() => {
+        global.fetch = TestHelper.getGlobalFetchMock();
+        return Onyx.clear().then(waitForBatchedUpdates);
+    });
+
+    describe('createWorkspace', () => {
+        it('creates a new workspace', async () => {
+            fetch.pause();
+            Onyx.set(ONYXKEYS.SESSION, {email: ESH_EMAIL, accountID: ESH_ACCOUNT_ID});
+            const policyID = Policy.generatePolicyID();
+            let adminReportID;
+            let announceReportID;
+            let expenseReportID;
+
+            await waitForBatchedUpdates();
+            Policy.createWorkspace(ESH_EMAIL, true, "Esh's Workspace", policyID);
+            await waitForBatchedUpdates();
+
+            let policy = await new Promise((resolve) => {
+                const connectionID = Onyx.connect({
+                    key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                    waitForCollectionCallback: true,
+                    callback: (workspace) => {
+                        Onyx.disconnect(connectionID);
+                        resolve(workspace);
+                    },
+                });
+            });
+
+            // check if policy was created with correct values
+            expect(policy.id).toBe(policyID);
+            expect(policy.name).toBe("Esh's Workspace");
```

Comment:
Suggestion: use a constant for the workspace name

Reply:
done here: https://github.com/Expensify/App/pull/34023/commits/9fc064608a78966b61f5db962316ee3fd1b4e2b9